data/json/recipes/recipe_deconstruction.json

```diff
@@ -2273,6 +2346,15 @@
     "qualities": [ { "id": "SCREW", "level": 1 } ],
     "components": [ [ [ "e_scrap", 2 ] ], [ [ "amplifier", 1 ] ], [ [ "scrap", 2 ] ] ]
   },
+  {
+    "result": "kettle",
+    "type": "uncraft",
+    "skill_used": "fabrication",
+    "difficulty": 1,
+    "time": "1 m 30 s",
+    "qualities": [ { "id": "SAW_M", "level": 1 } ],
+    "components": [ [ [ "sheet_metal_small", 3 ] ] ]
```

Comment:
Same issue with volume/weight - only 75ml of space and ~100ml of metal sheet.

Reply:
It seems roughly acurate. kettle weighs nearly 750g and is around 3 sheet_metal_small with 250g each.